pom.xml

```diff
@@ -50,18 +50,18 @@
         <dependencies>
             <dependency>
                 <groupId>ch.qos.logback</groupId>
-                <artifactId>logback-classic</artifactId>
+                <artifactId>logback-access</artifactId>
```

Comment:
I think the exclusive declaration of logback-access itself seems not required. Its coming anyway as transitive dependency from dropwizard-core.
![image](https://github.com/user-attachments/assets/08837c7c-ca79-4a71-ae1f-254dd3166b58)